### DISCUSSION TITLE
How about labeling (automatically/manually) failed runs as resolved (in UI)

### DISCUSSION THREAD

**Discussion by danthekishan** [dagster-io/dagster#22984] [Ideas]
It would be really awesome if we can label (automatically or manually) the failed runs as resolved or fixed.

For an example, when we encounter a failed run, most likely we are going to fix the code and re-execute it. When it runs successfully, it would be awesome to catagorize it as resolved/fixed without showing it in the failed tab. (probably based on the root run id). Then it will be so helpful for us track the failed runs.

**Comment by danielgafni** [comment]
Related: #22297
